api/test/docker/Dockerfile

```diff
@@ -23,8 +23,6 @@ COPY ./ ./
 
 RUN mkdir -p /go/manager-api/conf \
     && mkdir -p /go/manager-api/build-tools \
-    && go env -w GOPROXY=https://goproxy.io,direct \
```

Comment:
@nic-chen I was wondering that is removing Go proxy related to this PR ? Thanks.

Reply:
we don't need it in CI. @imjoey

Reply:
ping @imjoey

Reply:
> we don't need it in CI. @imjoey

@nic-chen thanks. It seems that we could also remove it in another PR. We'd better keep one PR only do one thing then. 😃 Thanks.